offchain/authority-claimer/Cargo.toml

```
@@ -12,9 +12,10 @@ test = false
 [dependencies]
 http-server = { path = "../http-server" }
 rollups-events = { path = "../rollups-events" }
+log = { path = "../log" }
```

Just keeping them in alphabetic order:
```suggestion
log = { path = "../log" }
rollups-events = { path = "../rollups-events" }
```

---

Done at 9e1b6f8e7bc477c334f4e109cf3bb7a1642bc44d